test/extensions/filters/http/compressor/compressor_filter_test.cc

```diff
@@ -812,6 +812,135 @@ TEST_F(MultipleFiltersTest, UseFirstRegisteredFilterWhenWildcard) {
   EXPECT_EQ(1, stats2_.counter("test2.compressor.test2.test.header_wildcard").value());
 }
 
+class ChooseFirstTest
+    : public MultipleFiltersTest,
+      public testing::WithParamInterface<std::tuple<std::string, bool, std::string>> {
+protected:
+  void SetUp() override {
+    envoy::extensions::filters::http::compressor::v3::Compressor compressor;
+    TestUtility::loadFromJson(R"EOF(
+{
+  "choose_first": "true",
```

Comment:
It would be great to test multiple filter as choose_first as true case also

Reply:
OK, I will add this next.

Reply:
I guess you forget to add test in the update, otherwise this PR LGTM

Reply:
Sorry I was on vacation before, I have added more test case, thanks!